.travis.yml

```diff
@@ -9,4 +9,7 @@ cache:
 services:
   - mongodb
 before_script:
-  - node server/seedDB.js
\ No newline at end of file
+  - node server/seedDB.js
+script:
+  - npm run test:lint
+  - npm run test
```

Comment:
Test calls lint so we don't need to have `run test:lint` in line 14 if we have this here.
test command: `"test": "concurrently 'npm run test:lint' 'npm run test:react' 'npm run test:api'",`

Reply:
We could have a separate test:all script to use concurrently. Doesn't the test:api need us to have our local db daemon up?

Reply:
Hmm I do think that having a separate lint call is important so lets pull the lint command out of test. Also, I do like the idea of creating a test all command, something meant to only test the src code logic -- `test:all`, `test:src`, `test:app`...
Thoughts @phuchle  @jwu910

Reply:
Sounds gucci.  I'll separate the test logic up and have travis run each one by one.